src/Component/Molecule/Contribution/Contribution.jsx

```diff
@@ -162,14 +148,19 @@ const Contribution = ({
       <summary>
         {summary}
       </summary>
-      <TagWrapper>
-        { tags.map((tag) => (
-          <Tag {...tag} />
-        ))}
+      <TagRow>
+        <TagWrapper>
+          { tags.map((tag) => (
+            <Tag>
```

Comment:
I believe this is necessary, I'm surprised eslint / react didn't flag it.
```suggestion
            <Tag key={tag}>
```